### PR TITLE
609 add remove earn groups to programme

### PR DESCRIPTION
**Pull request by RamyAlshurafa**
#### Issues this PR relates to

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

**npm run build:db**
---

#### Quick Pull Request Checklist
Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)
- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: server/modules/programmes/model/create.js [hunk 1]
```diff
@@ -53,10 +56,36 @@ const createProgramme = async ({
...
+    inviteToken,
   ]);
   return result.rows[0];
 };
 
+const createProgrammeEarnGroup = async (
+  { earnGroupUserId, programmeId, cratedBy },
+  client,
+) => {
+  const programmeData = [earnGroupUserId, programmeId, 'activated', cratedBy];
```
Comment (dupreesi) [inline server/modules/programmes/model/create.js:68]: createdBy

File: server/modules/programmes/use-cases/activate-deactivate-programme-earn-group.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  programmeId,
+  earnGroupUserId,
+  userId,
+  status,
+}) => {
+  if (status === 'activated') {
+    return Programme.createProgrammeEarnGroup({
+      earnGroupUserId,
+      programmeId,
+      cratedBy: userId,
```
Comment (dupreesi) [inline server/modules/programmes/use-cases/activate-deactivate-programme-earn-group.js:13]: createdBy

File: server/modules/programmes/model/create.js (hunk 1 above)
Comment (RamyAlshurafa) [reply server/modules/programmes/model/create.js:68]: done

File: server/modules/programmes/use-cases/activate-deactivate-programme-earn-group.js (hunk 2 above)
Comment (RamyAlshurafa) [reply server/modules/programmes/use-cases/activate-deactivate-programme-earn-group.js:13]: done

**Pull request merged**